pytorch_lightning/metrics/utils.py

```diff
@@ -14,38 +14,32 @@
 from typing import List, Optional
 
 import torch
-from torchmetrics.utilities.data import dim_zero_cat as __dim_zero_cat
-from torchmetrics.utilities.data import dim_zero_mean as __dim_zero_mean
-from torchmetrics.utilities.data import dim_zero_sum as __dim_zero_sum
-from torchmetrics.utilities.data import get_num_classes as __get_num_classes
-from torchmetrics.utilities.data import select_topk as __select_topk
-from torchmetrics.utilities.data import to_categorical as __to_categorical
-from torchmetrics.utilities.data import to_onehot as __to_onehot
-from torchmetrics.utilities.distributed import class_reduce as __class_reduce
-from torchmetrics.utilities.distributed import reduce as __reduce
+from torchmetrics.utilities.data import dim_zero_cat as _dim_zero_cat
+from torchmetrics.utilities.data import dim_zero_mean as _dim_zero_mean
+from torchmetrics.utilities.data import dim_zero_sum as _dim_zero_sum
+from torchmetrics.utilities.data import get_num_classes as _get_num_classes
+from torchmetrics.utilities.data import select_topk as _select_topk
+from torchmetrics.utilities.data import to_categorical as _to_categorical
+from torchmetrics.utilities.data import to_onehot as _to_onehot
+from torchmetrics.utilities.distributed import class_reduce as _class_reduce
+from torchmetrics.utilities.distributed import reduce as _reduce
```

Comment:
just for the record, this was the problem distributed in combination with wrapper...
cc: @awaelchli

Reply:
I'm not sure I understand
what do these import statements have to do with distributed?

Reply:
as it was imported with __ there was some problem with passing methods to the device, to be honest not very sure what happened... :/